src/main/scala/it/pagopa/interop/attributeregistryprocess/server/impl/Dependencies.scala

```diff
@@ -0,0 +1,141 @@
+package it.pagopa.interop.attributeregistryprocess.server.impl
+
+import akka.actor.typed.ActorSystem
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.directives.SecurityDirectives
+import com.atlassian.oai.validator.report.ValidationReport
+import com.nimbusds.jose.proc.SecurityContext
+import com.nimbusds.jwt.proc.DefaultJWTClaimsVerifier
+import it.pagopa.interop.commons.cqrs.service.{MongoDbReadModelService, ReadModelService}
+import it.pagopa.interop.commons.jwt.service.JWTReader
+import it.pagopa.interop.commons.jwt.service.impl.{DefaultJWTReader, getClaimsVerifier}
+import it.pagopa.interop.commons.jwt.{JWTConfiguration, KID, PublicKeysHolder, SerializedKey}
+import it.pagopa.interop.commons.utils.TypeConversions._
+import it.pagopa.interop.commons.utils.errors.{Problem => CommonProblem}
+import it.pagopa.interop.commons.utils.service.{OffsetDateTimeSupplier, UUIDSupplier}
+import it.pagopa.interop.commons.utils.{AkkaUtils, OpenapiUtils}
+import it.pagopa.interop.attributeregistryprocess.api.impl.{
+  AttributeRegistryApiMarshallerImpl,
+  AttributeRegistryApiServiceImpl,
+  HealthApiMarshallerImpl,
+  HealthServiceApiImpl
+}
+import it.pagopa.interop.attributeregistryprocess.api.{AttributeApi, HealthApi}
+import it.pagopa.interop.attributeregistryprocess.common.system.ApplicationConfiguration
+import it.pagopa.interop.attributeregistryprocess.error.ResponseHandlers.serviceCode
+import it.pagopa.interop.attributeregistryprocess.service.impl._
+import it.pagopa.interop.attributeregistryprocess.service._
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+import com.typesafe.scalalogging.{Logger, LoggerTakingImplicit}
+import it.pagopa.interop.commons.logging.{CanLogContextFields, ContextFieldsToLog}
+
+trait Dependencies {
+
+  implicit val loggerTI: LoggerTakingImplicit[ContextFieldsToLog] =
+    Logger.takingImplicit[ContextFieldsToLog]("OAuth2JWTValidatorAsContexts")
+
+  val uuidSupplier: UUIDSupplier               = UUIDSupplier
+  val dateTimeSupplier: OffsetDateTimeSupplier = OffsetDateTimeSupplier
+
+  def jwtValidator(): Future[JWTReader] = JWTConfiguration.jwtReader
+    .loadKeyset()
+    .map(keyset =>
+      new DefaultJWTReader with PublicKeysHolder {
+        var publicKeyset: Map[KID, SerializedKey]                                        = keyset
+        override protected val claimsVerifier: DefaultJWTClaimsVerifier[SecurityContext] =
+          getClaimsVerifier(audience = ApplicationConfiguration.jwtAudience)
+      }
+    )
+    .toFuture
+
+  val readModelService: ReadModelService = new MongoDbReadModelService(ApplicationConfiguration.readModelConfig)
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135226074)